automatic/vscode-insiders/vscode-insiders.nuspec

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-insiders</id>
+    <title>Visual Studio Code Insiders</title>
+    <version>1.46.0-insider</version>
```

Comment:
Just to note here as well, it can not be set to the latest version as it will prevent the updater to automatically update the package.